parser-typechecker/tests/Unison/Test/ANF.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE BangPatterns #-}
 {-# LANGUAGE PatternGuards #-}
+{-# LANGUAGE TypeApplications #-}
```

Comment:
Is this not yet in our package defaults? We should add it :)

Reply:
Good point — it looks like it is already in our package defaults, and I probably just added it here in a desperate attempt to make HLS work better in some moment. This will probably keep happening though until I have the list of package defaults memorized, and stop trying such things.

Also it looks like there are lots of existing spots where we explicitly enable extensions that are already in our package defaults, so we should make a PR for that, assuming it doesn't break some standalone ormolu that we've decided we depend on and which isn't aware of package defaults. 😬